docs/getting_started.md

```diff
@@ -1,19 +1,21 @@
 # Getting started
 
 ## Using Vertex
-[Vertex](https://search.asf.alaska.edu/#/), ASF's data search portal, is the primary method for using HyP3,
-It allows you to request jobs and view previous jobs with a friendly interface. For more information
-check out [HyP3 in Vertex](using/vertex.md).
+
+[Vertex](https://search.asf.alaska.edu/), ASF's data search portal, is the easiest way to use HyP3. Vertex provides a
+friendly interface to request jobs and review previous jobs. Visit [Using HyP3: Vertex](using/vertex.md) to learn more.
 
 ## Using the Python SDK
-The HyP3 SDK is a Python library intended for people who want to use HyP3 programmatically.
-It is available in [PyPI](https://pypi.org/project/hyp3-sdk/) and can be installed with `pip install hyp3-sdk`.
-Check out [Using the HyP3 SDK](using/skd.md) for more information.
+
+The HyP3 SDK is a Python library for using HyP3 programmatically. It is available on
+[PyPI](https://pypi.org/project/hyp3-sdk/) and can be installed with `pip install hyp3-sdk`. Visit
+[Using HyP3: SDK](using/skd.md) to learn more.
 
 !!! Warning
```

Comment:
I think I meant for these to be hashes, it doesn't render anything and looks out of place